erpnext/public/js/controllers/taxes_and_totals.js

```
@@ -822,50 +930,63 @@
 			let base_grand_total = this.frm.doc.base_rounded_total || this.frm.doc.base_grand_total;
 
-			if(this.frm.doc.party_account_currency == this.frm.doc.currency) {
-				var total_amount_to_pay = flt((grand_total - this.frm.doc.total_advance
-					- this.frm.doc.write_off_amount), precision("grand_total"));
+			if (this.frm.doc.party_account_currency == this.frm.doc.currency) {
+				var total_amount_to_pay = flt(
+					grand_total - this.frm.doc.total_advance - this.frm.doc.write_off_amount,
+					precision("grand_total")
+				);
 			} else {
 				var total_amount_to_pay = flt(
```

Biome (2.1.2)

[error] Shouldn't redeclare 'total_amount_to_pay'. Consider to delete it or rename it.

'total_amount_to_pay' is defined here:


(lint/suspicious/noRedeclare)